src/include/wiredtiger.in

```diff
@@ -1693,7 +1693,8 @@ struct __wt_session {
 	 * @exclusive
 	 *
 	 * @param session the session handle
-	 * @param name the URI of the table or file to verify
+	 * @param name the URI of the table or file to verify, optional if verifying the history
```

Comment:
Note: We get back an error if URI is provided with "history_store" verification.